assets/scripts/info_bubble/info_bubble.js

```diff
@@ -38,7 +38,6 @@ import {
   processWidthInput
 } from '../util/width_units'
 import { isAnyMenuVisible, hideAllMenus } from '../menus/menu_controller'
-import { registerKeypress } from '../app/keypress'
```

Comment:
See comment in `keyboard_commands.js` re: whether keypress registration should remain here.